lib/Reference/ReferenceHelper.php

```diff
@@ -43,8 +49,8 @@ public function matchReference(string $referenceText): bool {
 
 		// link example:
 		// https://nextcloud.local/apps/tables/#/table/3
-		$noIndexMatch = preg_match('/^' . preg_quote($start, '/') . '\/#\/table\/\d+$/i', $referenceText) === 1;
-		$indexMatch = preg_match('/^' . preg_quote($startIndex, '/') . '\/#\/table\/\d+$/i', $referenceText) === 1;
+		$noIndexMatch = preg_match('/^' . preg_quote($start, '/') . '\/#\/view\/\d+$/i', $referenceText) === 1;
+		$indexMatch = preg_match('/^' . preg_quote($startIndex, '/') . '\/#\/view\/\d+$/i', $referenceText) === 1;
```

Comment:
This would break existing links, but might be acceptable considering still pre 1.0 ;)

Reply:
The old links now work and redirect to the corresponding base view. The widgets from the smart picker won't work but I think that is as you said.